src/chat/chat_panel.rs

```diff
@@ -762,7 +766,7 @@ impl ChatPanel {
         }
 
         // Let's confirm we're in an appropriate state to send a message
-        self.update_state(store);
+        self.update_state(scope);
```

Comment:
This call and the if should be technically unnecessary. Like, how did you trigger a `send_message` if you are at a state where the UI should be disabled? If we can, there is a bug.

But yeah, let's just adapt the code as necessary and preserve it just in case.

If we do work about making chats embedded, this may be re-implemented anyways.